src/Microsoft.Dnx.Tooling/Packages/Add/AddOptions.cs

```diff
@@ -9,5 +9,9 @@ namespace Microsoft.Dnx.Tooling.Packages
     public class AddOptions : PackagesOptions
     {
         public string NuGetPackage { get; set; }
+
+        // When "dnu packages add" is invoked as a subcommand of other commands
+        // this option can be set to true to have trimmed output
+        public bool ConciseOutput { get; set; }
```

Comment:
I am blocking part of output here and that's why I don't name it suppress. Sounds like you are OK with blocking all output. Then I'll block all output by making `packages add` quiet. There is even no new option needed here.